### PR TITLE
feat: add eval command

### PR DESCRIPTION
**Pull request by jacc**
Added an extremely useful eval command that totally runs code and doesn't return a video link.

### CODE REVIEW

File: packages/aquarius/src/bot/commands/eval.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { randomValue } from '../../core/helpers/lists';
+import getLogger, { getMessageMeta } from '../../core/logging/log';
+
+const log = getLogger('eval');
+
+/** @type {import('../../typedefs').CommandInfo} */
+export const info = {
+  name: 'eval',
+  description: 'Evaluate arbitrary code.',
+  usage: '```@Aquarius eval```',
```
Comment (IanMitchell) [inline packages/aquarius/src/bot/commands/eval.js:10]: `@Aquarius eval <input>`

File: packages/aquarius/src/bot/commands/eval.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { randomValue } from '../../core/helpers/lists';
```
Comment (IanMitchell) [inline packages/aquarius/src/bot/commands/eval.js:1]: Del

Comment (jacc) [reply packages/aquarius/src/bot/commands/eval.js]: resolved

File: packages/aquarius/src/bot/commands/eval.js (hunk 1 above)
Comment (jacc) [reply packages/aquarius/src/bot/commands/eval.js]: resolved

**Comment by cnrad** [comment]
image

**Pull request closed without merging**